static_src/stores/user_store.js

```diff
@@ -155,13 +166,15 @@ class UserStore extends BaseStore {
       }
 
       case userActionTypes.USER_DELETED: {
-        const deleted = this.get(action.userGuid);
-        if (deleted) {
-          const index = this._data.indexOf(deleted);
-          this._data.splice(index, 1);
-          this._error = null;
+        const index = this._data.findIndex((d) =>
```

Comment:
Great, I'll implement that. I think that it makes sense as just `delete` because `get` uses `guid` as the index so it will basically be the same convention